imports/ui/components/dashboard/dashboard_cards_section.jsx

```diff
@@ -1,21 +1,29 @@
 import React, {Component} from 'react'
-import {CardDeck, Card, Collapse} from 'react-bootstrap'
+import {CardColumns, Card, Collapse} from 'react-bootstrap'
 import MatchCard from './match_card'
 
 class DashboardCardsSection extends Component {
   render () {
-    const {section, visibility} = this.props
-
+    const {section, visibility, entries} = this.props
+    //console.log(this.props.entries)
+    const sectionMapper = {
+      'Mentors': 'Mentees to help', 
+      'Mentees': 'Mentors',
+      'OSS Projects':'OSS Contributors to their project',
+      'OSS Contributors': 'OSS Projects to contribute to',
+      'Accountabili-buddies': 'Accountabili-buddies',
+      'Other':'Other'
+    }
     return (
       <Collapse in={visibility}>
-        <Card as='section' className='mb-4'>
+        <Card as='section'>
           <Card.Header className='text-capitalize'>
-            People looking for {section}
-          </Card.Header>
-          <Card.Body>
-              <CardDeck className='d-block'>
-                <MatchCard name="Test" overview='some text'/>
-              </CardDeck>					
+            People looking for {sectionMapper[section]}
+          </Card.Header>  
+          <Card.Body as={CardColumns}>
+               {entries.length ? entries.map((entry, i) => (
```

Comment:
I think we should prefer map functions as separate methods instead of evaluating them inside the `return`. I think they are much more readable this way: 

```jsx
render () {
   const { section, ... } = this.props
   const sectionMapper = { ... }

   const matchCards = () => {
      if (entries.length > 0) {
         return entries.map((entry, i) => {
             <MatchCard key={i} lookingFor ... />
         })
      } else {
         return <Card.Text>No active entries</Card.Text>
      }
   }

   return (
       ...
          <Card.Body as={CardColumns}>
              {matchCards}
          </Card.Body>
       ...
   )
}
```

Reply:
Additionally, AirBnB had a good post on not using indexes as keys: 
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Thanks for the article! What do you think about using `entry._id` as the key?

Reply:
I'm not convinced that ```const matchCards = () => {
      if (entries.length > 0) {
         return entries.map((entry, i) => {
             <MatchCard key={i} lookingFor ... />
         })
      } else {
         return <Card.Text>No active entries</Card.Text>
      }
   }``` is a lot more readable than using a ternary operator, which seems to be standard in React, though.

Reply:
So long as `entry._id` is actually unique to that `entry` and not its index placement in the array 👍

Reply:
Re: ternary operators -- I'm not fully convinced it's that much of a standard either 😅 It just seems like bad practice and a lot of examples has just normalized it. Ternary operators are really more meant for a single line evaluation. 

From the [docs](https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator): 
> It can also be used for larger expressions although it is less obvious what’s going on

In any case, the main goal is to ensure that our code is readable and easy to understand regardless of how much React experience a contributor has. It would be much better to write out some proper if/else statements as it's clear exactly what's happening. 

(See example from: 
https://reactjs.org/docs/conditional-rendering.html#element-variables
https://codepen.io/gaearon/pen/QKzAgB?editors=0010)